docs/casa/bom/printing_guide.md

```diff
@@ -20,16 +20,16 @@ Casa uses 3 different print profiles. Each profile is designed around a 0.4 mm n
 
 ## File Naming
 
-With your STL files downloaded and your printer warmed up, you may be wondering which parts to print in your favorite colors? Have no fear, we've got your back - each file is labelled.
+With your STL files downloaded and your printer warmed up, you may be wondering which parts to print in your favorite colors. Have no fear, we've got your back - each file is labeled.
```

Comment:
`labeled` is an Americanisation of `labelled`. But then we use `colors` above instead of `colours` so I dunno who wrote that (probably me! :facepalm: ).

```suggestion
With your STL files downloaded and your printer warmed up, you may be wondering which parts to print in your favorite colours. Have no fear, we've got your back - each file is labelled.
```

I'd prefer we stick to British English spellings of... contentious words if possible.